nifi-nar-bundles/nifi-dropbox-bundle/nifi-dropbox-processors/src/test/java/org/apache/nifi/processors/dropbox/PutDropboxTest.java

```diff
@@ -255,18 +258,18 @@ void testFileUploadLargeFile() throws Exception {
                 .thenReturn(mockUploadSessionAppendV2Uploader);
 
         //finish session: 30 - 8 - 2 * 8 = 6 bytes uploaded
-        CommitInfo commitInfo = CommitInfo.newBuilder(getPath(TEST_FOLDER , FILENAME_1))
+        CommitInfo commitInfo = CommitInfo.newBuilder(getPath(TEST_FOLDER, FILENAME_1))
```

Comment:
I don't feel strongly, but I wouldn't recreate this object solely to have a prototype that can be used as a matcher. Plain any() matcher would be enough for me, but if we wanted to match its properties, I would create a Factory, especially for CommitInfo, which we can mock adequately.